### DISCUSSION TITLE
Groq API request within gpt-pilot

### DISCUSSION THREAD

**Discussion by DrewPear309** [Pythagora-io/gpt-pilot#761] [Q&A]
Howdy,
What a great bit of code. Thank you.
I'm trying to use the Groq API and I'm receiving an error.
it appears that the API endpoint call is truncating the API URL perhaps.
The full endpoint is, https://api.groq.com/openai/v1 
I've changed the model used to llama2-70b.
In the env file I've only changed the endpoint, the API key, and the model name
Any thoughts? Thank you!

ERROR:
There was a problem with request to openai API:
API responded with status code: 404. Request token size: 23 tokens. Response text: {"error":{"message":"Unknown request URL: POST /openai/v1. Please check the URL for typos, or see the docs at https://console.groq.com/docs/","type":"invalid_request_error","code":"unknown_url"}}

**Comment by yernero** [comment]
Im pretty sure you need a completions endpoint.

**Comment by timoleiserconsulting** [comment]
No the Endpoint you need is: https://api.groq.com/v1/chat/completions
